sgv2-restapi/src/test/java/io/stargate/sgv2/restsvc/grpc/ToProtoConverterTest.java

```diff
@@ -24,21 +25,22 @@
     return new Arguments[] {
       arguments(123, basicType(TypeSpec.Basic.INT), Values.of(123)),
       arguments(-4567L, basicType(TypeSpec.Basic.BIGINT), Values.of(-4567L)),
-      arguments("abc", basicType(TypeSpec.Basic.TEXT), Values.of("abc")),
+      arguments("abc", basicType(TypeSpec.Basic.VARCHAR), Values.of("abc")),
```

Comment:
Unrelated change, to remove warnings: `Basic.TEXT` deprecated should (have) use(d) `Basic.VARCHAR` already